.travis.yml

```
@@ -19,6 +19,7 @@ env:
   - LAYER=igbinary PHP="72 73 74"
   - LAYER=imagick PHP="72 73 74"
   - LAYER=mailparse PHP="72 73 74"
+  - LAYER=odb-snowfake PHP="72 73 74"
```

```suggestion
  - LAYER=odbc-snowflake PHP="72 73 74"
```

I just quickly looked over it but naming the layer correctly might help already ;)

---

Oops. sorry =)

Thank you for your input.